css-positioning/relative.css

```diff
@@ -0,0 +1,36 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
```

Comment:
better to put position relative here so that it doesn't have to be placed on each image separately.